includes/Checker/CLI_Runner.php

```diff
@@ -32,15 +33,7 @@ class CLI_Runner extends Abstract_Check_Runner {
 	 * @return bool Returns true if is an CLI request for the plugin check else false.
 	 */
 	public function is_plugin_check() {
```

Comment:
See my other comment, I think making this method static makes more sense than introducing a similar method somewhere else that is not specific to CLI.

It's perfectly reasonable for this method to be static since it only checks something in the global scope and has nothing to do with an instance of this class.

Reply:
@mukeshpanchal27 @felixarntz should we also update the `AJAX_Runner` method to also be static for consistency? We would also need to update the `Plugin_Request_Utility::initialize_runner` method to use that static methods also.